### ISSUE TITLE
A Voice-activity-detector wav2label model as a built in model

### ISSUE THREAD

**Issue by bryan-pardo** [audacitorch/audacity#64]
**Is your feature request related to a problem? Please describe.**
Blind users really need to remove silence in voice tracks and have a hard time finding the silence without listening for it. 

**Describe the solution you'd like**
A voice-activity-detector wav2label model would help a lot.

**Describe alternatives you've considered**
There is no alternative!

**Additional context**
This would also help sighted users quite a bit.
